admin/app/components/solidus_admin/sidebar/item/component.rb

```diff
@@ -43,7 +43,9 @@ def link_level_classes
   def nested_items
     return unless @item.children?
 
-    tag.nav do
+    tag.nav(
+      class: "hidden group-hover:block"
```

Comment:
hahah sorry, I left that part out! 😅 

In my interpretation there was not expansion on hover, only on click, and if that's the case the implementation will be simpler. I think I didn't think of expanding on hover also because it might be tricky to navigate, with things moving under the pointer. 

That's why I was suggesting a check with a designer.

Reply:
Oh, I see. Will check :slightly_smiling_face:

Reply:
You were right! I updated the code accordingly.